invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/RenameCommunitySlugButton.js

```diff
@@ -90,7 +90,8 @@ export class RenameCommunitySlugButton extends Component {
           <Modal.Content>
             <Form onSubmit={this.handleChange}>
               <Form.Input
-                label={i18next.t("Enter the new unique identifier of the community.")}
+                label={i18next.t("New unique identifier of the community")}
+                placeholder={i18next.t("New unique identifier of the community")}
```

Comment:
could you show us how does it look like (screenshot)? I am not sure if the field label should be the same as a placeholder

Reply:
![image](https://user-images.githubusercontent.com/74763324/175248156-3d4e24f1-7e3a-4168-b4bd-413b58d1b671.png)

Popup shown when clicking here on "Change identifier"
![image](https://user-images.githubusercontent.com/74763324/175248262-1886f24b-b7ff-4679-bb84-a69b8101e1a3.png)

Reply:
I thought it might be unique to other places like this:
![image](https://user-images.githubusercontent.com/74763324/175248524-3c7a70e9-b883-44c7-a0de-18934051fd5a.png)
and this
![image](https://user-images.githubusercontent.com/74763324/175248555-398374a4-3aff-4d52-86fa-00431e2bd6a1.png)
however, both of them aren't pop-ups.